lib/src/native/realm_core.dart

```diff
@@ -384,6 +384,13 @@ class _RealmCore {
     });
   }
 
+  void realmListDeleteAll(RealmList list) {
```

Comment:
are we deleting or removing items from the list. I guess we ment `deleting` is when we delte objects from the realm but when we `remove` we just remove them from the list and objects stay in the realm. So I guess this should be `realmListRemoveAll`